### PR TITLE
fix: update LDC form mappings

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Changes directly correspond to feedback reported via linked Trello tickets, most applies to LDC-E but I updated LDC-P template for consistency wherever possible.

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[(POs to re-review) Generate an application form](https://trello.com/c/FK63YOsa/2229-pos-to-re-review-generate-an-application-form)

**Comment by jessicamcinchak** [comment]
[Change application forms so they include the description of development](https://trello.com/c/WwJlaW4Z/2660-change-application-forms-so-they-include-the-description-of-development)

File: src/templates/docx/LDCETemplate.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export function LDCETemplate(passport: Passport): Document {
         fields: [
           {
             name: "Is there an agent?",
-            value: getBoolean("applicant.agent.form") ? "Yes" : "No",
+            value: get("applicant.agent.form"),
```
Comment (jessicamcinchak) [inline src/templates/docx/LDCETemplate.ts:49]: This is just a "Yes" / "No" SetValue, no need for `getBoolean` here - see https://editor.planx.uk/opensystemslab/about-the-applicant

File: src/templates/docx/LDCETemplate.ts [hunk 2]
```diff
@@ -195,7 +197,13 @@ export function LDCETemplate(passport: Passport): Document {
...
           {
             name: "Please state under what grounds is the certificate sought",
-            value: get("application.basisOfLawfulness"),
+            value:
+              get("application.basisOfLawfulness") === "Immune"
+                ? `Immune - The property has been in this use since ${
+                    get("proposal.start.date") ||
+                    get("proposal.completion.data")
+                  }`
+                : get("application.basisOfLawfulness"),
```
Comment (jessicamcinchak) [inline src/templates/docx/LDCETemplate.ts:206]: This was the trickiest point of feedback - see thread here: https://opensystemslab.slack.com/archives/C5Q59R3HB/p1698851967728249

I think this is a decent first solution, but if this comes back up then we expect to raise a rather time intensive green content ticket to update `application.basisOfLawfulness` values to better match Secretary of State options set.

File: src/templates/docx/LDCETemplate.ts [hunk 3]
```diff
@@ -74,7 +74,9 @@ export function LDCETemplate(passport: Passport): Document {
         fields: [
           {
             name: "Is the applicant’s address the same as the site address?",
-            value: get("applicant.sameAddress.form"),
+            value:
+              get("applicant.sameAddress.form") ||
+              get("applicant.address.sameAsSiteAddress"),
```
Comment (jessicamcinchak) [inline src/templates/docx/LDCETemplate.ts:79]: This is written into content quite strangely, and was failing to pick up "No" before https://editor.planx.uk/opensystemslab/about-the-applicant
Screenshot from 2023-11-02 15-02-52

File: src/templates/docx/LDCETemplate.ts (hunk 2 above)
Comment (DafyddLlyr) [reply src/templates/docx/LDCETemplate.ts:206]: This makes sense to me!

**Review by DafyddLlyr — APPROVED** [review]
Great to see these ones picked up and fixed 🙌

**Pull request merged**